src/main/java/core/mate/academy/model/BulldozerProducer.java

```diff
@@ -0,0 +1,17 @@
+package core.mate.academy.model;
+
+import core.mate.academy.service.MachineProducer;
+import java.util.ArrayList;
+import java.util.List;
+
+public class BulldozerProducer implements MachineProducer<Bulldozer> {
+
+    @Override
+    public List<Bulldozer> get() {
+        List<Bulldozer> listBuldozers = new ArrayList<>();
+        listBuldozers.add(new Bulldozer());
+        listBuldozers.add(new Bulldozer());
+        return listBuldozers;
+    }
```

Comment:
better way use special methods for create Bulldozer and inizialize listBuldozers and get result in this method

Reply:
same for Truck and Excavator

Reply:
done